metrics-pusher/README.md

```diff
@@ -5,17 +5,20 @@
 Run using docker:
 
 ```sh
-docker run -v /var/run/docker.sock:/var/run/docker.sock \
+docker run --privileged \
```

Comment:
Are there options to manage allow the capabilities more fine-grained than "privileged"?

Reply:
Changed to `--cap-add "CAP_SYS_PTRACE"`, but is not always necessary (see README)